homeassistant/components/shopping_list.py

```diff
@@ -201,7 +192,7 @@ class CreateShoppingListItemView(http.HomeAssistantView):
     url = '/api/shopping_list/item'
     name = "api:shopping_list:item"
 
-    @RequestDataValidator(vol.Schema({
+    @http.RequestDataValidator(vol.Schema({
```

Comment:
Why did you revert this? The file has been moved to its own file.

Reply:
I'm not sure how that happened. It definitely wasn't on purpose. I'll revert the accidental revert.